udata_gouvfr/theme/templates/post/list.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'post/list.html' %}
+
+{% set a_link = config['POST_BANNER_LINK'] %}
+{% set a_content = config['POST_BANNER_MESSAGE'] %}
+
+{% block alert_section %}
+{% if config['POST_BANNER_ACTIVATED'] %}
+<div class="container-fluid alert-info alert">
+    <div class="container">
```

Comment:
keep the role from here https://github.com/etalab/udata-gouvfr/pull/512/commits/8b7f83ca1d1e750e01006f17073def4979cd0bff#diff-a7db4438233ee9ea6df522d198260ba9dc57f978809ab6fd144399fda794daffL9